<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>659</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QDockWidget" name="InputDock">
        <property name="minimumSize">
         <size>
          <width>230</width>
          <height>228</height>
         </size>
        </property>
        <widget class="QWidget" name="dockWidgetContents_2">
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="ZoneLabel">
            <property name="text">
             <string>Select Zone</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="ZoneCombo"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="HazardLabel">
            <property name="text">
             <string>Select Hazard</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="HazardCombo"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="LevelLabel">
            <property name="text">
             <string>Level</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="LevelCombo">
            <item>
             <property name="text">
              <string>Upazilla</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Village</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="UpazillaLabel">
            <property name="text">
             <string>Upazilla Name</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QPushButton" name="CalculatePButton">
            <property name="text">
             <string>Calculation</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QPushButton" name="MapPBtn">
            <property name="text">
             <string>Save Map</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="UpazillaCombo"/>
          </item>
          <item row="5" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="4" column="1">
           <widget class="QComboBox" name="VillageCombo"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="VillageLabel">
            <property name="text">
             <string>Village Name</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Adaptation Deficiency</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDockWidget" name="ImageDock">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>400</height>
         </size>
        </property>
        <widget class="QWidget" name="dockWidgetContents">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="ImView">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">Background : rgb(255, 255, 255)</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QDockWidget" name="TableDock">
        <widget class="QWidget" name="dockWidgetContents_3">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QTableWidget" name="Table1"/>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="Table1PBtn">
            <property name="text">
             <string>Save Table</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="Table2PBtn">
            <property name="text">
             <string>Save Table</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QTableWidget" name="Table2"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
